modules/createbooks.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable */
+// import { Render } from "./render.js";
+
+export class Storage {
+  static storeData() {
+    localStorage.setItem('bookList', JSON.stringify(bookList));
+  }
+
+  static initiateBooksData() {
+    const initialBooks = localStorage.getItem('bookList');
+    if (initialBooks) {
+      return JSON.parse(initialBooks);
+    }
+    return [];
+  }
+}
+
+export let bookList = Storage.initiateBooksData();
```

Comment:
- [ ] Kindly make this a module and import it into this file. Having several modules makes it easy to rea and maintain codes. 👍